homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -69,6 +69,8 @@ class PanasonicVieraTVEntity(MediaPlayerEntity):
         | MediaPlayerEntityFeature.STOP
         | MediaPlayerEntityFeature.BROWSE_MEDIA
     )
+    _attr_has_entity_name = True
+    _attr_name = None
```

Comment:
The problem is, that for this integration, `device_info` can be missing.

Reply:
I just also checked, and apparently there already is a migration added to add device_info in #41028. This was added 3 years ago. Do we still expect users to not have this setup?

Reply:
I agree, but... that is the current change matches the current codebase. Let's clean in up in a followup PR.